parsl/executors/status_handling.py

```diff
@@ -167,10 +167,14 @@ def tasks(self) -> Dict[object, Future]:
     def provider(self):
         return self._provider
 
-    def _filter_scale_in_ids(self, to_kill, killed):
+    def _filter_scale_in_ids(self, to_kill: Sequence[Any], killed: Sequence[bool]) -> Sequence[Any]:
         """ Filter out job id's that were not killed
         """
         assert len(to_kill) == len(killed)
+
+        if False in killed:
+            logger.warning(f"Some jobs were not killed successfully: to_kill list is {to_kill}, killed list is {killed}")
```

Comment:
Are `to_kill` and `killed` `zip()`pable?  If so, consider spelling out the ones that succeeded and failed.  Something like:
```python
if False in killed:
    s = (j for j, s in zip(to_kill, killed) if s)  # "success"
    f = (j for j, s in zip(to_kill, killed) if not s)  # "fail"
    logger.warning(
      "Failed to stop all jobs."
      "\n  Successfully stopped: {', '.join(s)}"
      "\n        Failed to stop: {', '.join(f)}"
    )
```
Format as you will, of course, the but the point is to save the user some log-trawling thinking but spelling it out just a smidgen more.

Reply:
yeah that is a better way to present the info. I ended up not trying to do join style formatting because it looks weird when the success list is empty, and rather than going down a rat-hole of dealing with that case, I let Python's default list rendering do it.